import-issues.py

```diff
@@ -313,17 +331,19 @@ def main():
                 break
 
         else:
-            labels_log.info("Creating label %s", label_name)
-            label_req = gh_request(
-                'repos/%s/%s/labels' % (args.github_owner,
-                                        args.github_repo),
-                 args,
-                 data={'name': label_name,
-                       'color': LABEL_COLORS[label_color]
-                       },
-            )
-            if not label_req:
-                sys.exit(1)
+            if not dryrun:
+                labels_log.info("Creating label %s", label_name)
```

Comment:
i think these logs would be better outside the dryrun conditional. that way, there's more info about what would happen